Client/src/Modelo/Torneo.java

```diff
@@ -0,0 +1,62 @@
+package Modelo;
+
+import java.util.ArrayList;
+
+/**
+ * @author Grupo
+ * 
+ *         Esta clase representa el funcionamiento de un torneo eliminatorio
+ *         para 8 competidores. Cuenta con el conjunto de entrenadores que
+ *         participaran en dicho torneo, el nombre del torneo y una
+ *         variable que hara referencia a una sola instancia del
+ *         torneo.
+ */
+public class Torneo
+{
+	private String nombre_torneo;
+	private ArrayList<Entrenador> entrenadores;
+	private static Torneo instance_torneo = null;
+	private ArrayList<Ronda> rondas;
+	
+	/**
+	 * Constructor para iniciar la lista que contendra a los entrenadores.
+	 */
+	private Torneo()
+	{
+		this.entrenadores = new ArrayList<Entrenador>();
+	}
+
+	/**
+	 * Este m�todo nos devuelve una instancia �nica del torneo.
+	 * 
+	 * @return Referencia de tipo Torneo
+	 */
+	public static Torneo getInstance()
+	{
+		if (Torneo.instance_torneo == null)
+			Torneo.instance_torneo = new Torneo();
+		return instance_torneo;
+	}
+
+	public String getNombre_torneo()
+	{
+		return nombre_torneo;
+	}
+
+	public void setNombre_torneo(String nombre_torneo)
+	{
+		this.nombre_torneo = nombre_torneo;
+	}
+
+	/**
+	 * Estem m�todo se encarga de agregar un entrenador a la lista de los mismos.
+	 */
+	public void AgregaEntrenador(Entrenador entrenador)
```

Comment:
```suggestion
	public void agregaEntrenador(Entrenador entrenador)
```
Para respetar la nomenclatura determinada de los métodos en Java, deben comenzar siempre en minúscula, sin separaciones y cada palabra se diferencia con mayúscula